default/stringtables/en.txt

```diff
@@ -12177,6 +12491,16 @@ BLD_IMPERIAL_PALACE_DESC
 
 Represents imperial power and prestige and functions as a center of control for the empire's holdings.'''
 
+BLD_IMPERIAL_GARDENS
+Imperial Gardens
+
+BLD_IMPERIAL_GARDENS_DESC
+'''Extensive gardens around the palace, in a romantic or cosy or magnificent style, depending on the tastes of the local population.  Genetic engineering is employed to make many of the plants look even nicer.  Like in most capital cities, people don't complain about a higher population density as long as some parts of the city are sufficiently beautiful.
+
+Increases Population capacity by planet size on the planet where it is built: ([[SZ_TINY]] +2) ([[SZ_SMALL]] +4) ([[SZ_MEDIUM]] +6) ([[SZ_LARGE]] +8) ([[SZ_HUGE]] +10).
+
+It can only by built around the [[buildingtype BLD_IMPERIAL_PALACE]], and only if the empire doesn't have [[BLD_IMPERIAL_GARDENS]] yet.
+
```

Comment:
missing closing `'''` which leads to the Imperial Gardens article having a bunch of broken additional stringtable stuff in the pedia.